modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormPagesTemplateContextFactory.java

```diff
@@ -169,6 +171,19 @@ private List<Object> _createColumnsTemplateContext(
 		List<Object> columnsTemplateContext = new ArrayList<>();
 
 		for (DDMFormLayoutColumn ddmFormLayoutColumn : ddmFormLayoutColumns) {
+			if (!GetterUtil.getBoolean(
+					PropsUtil.get("feature.flag.LPS-164998"))) {
+
+				List<String> ddmFormFieldNames =
```

Comment:
Please, declare this variable before the IF because we can write only 1 IF:
` !GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-164998")) && ddmFormFieldNames.contains("limitToOneSubmissionPerUser")`

What do you think?

Reply:
make sense